ocamltest/builtin_actions.ml

```diff
@@ -110,7 +110,7 @@ let not_windows = make
 
 let is_bsd_system s =
   match s with
-  | "bsd_elf" | "netbsd" | "freebsd" | "openbsd" -> true
+  | "bsd_elf" | "netbsd" | "freebsd" | "openbsd" | "cygwin" -> true
```

Comment:
That's a bold claim.  In tests where it matters, could we test "is_bsd_system or is_cygwin" instead?

Reply:
Oh, OK, I see this is a temporary commit.